analysis/estimate/notebooks/compare-estimate-consensus.Rmd

```diff
@@ -0,0 +1,340 @@
+* Only SCPCP000006 and SCPCP000009 have samples with truly visible levels of "stromal cells", particularly SCPCP000009
+  * It is more challenging in these cancer types (wilms and osteo, respectively) to distinguish among stromal and cancer cells get, so the stromal proportions here are not necessarily definitive
+
+## Relationship between cell types and pseudobulk ESTIMATE scores
+
+Next, we'll directly look at the relationships between fractions of these cell type groups and ESTIMATE scores.
+We'll do this now with pseudobulk ESTIMATE scores, which is from the same underlying modality as the consensus cell types.
+
+```{r}
+wide_category_df <- celltype_category_df |>
+  dplyr::mutate(estimate_category = glue::glue("celltype_{estimate_category}")) |>
+  tidyr::pivot_wider(
+    names_from = estimate_category, 
+    values_from = frac, 
+    values_fill = 0
+  )
+  
+  
+combined_wide_df <- estimate_df |>
+  dplyr::filter(expression_type == "pseudobulk") |>
+  dplyr::select(-expression_type) |>
+  dplyr::mutate(score_type = glue::glue("estimate_score_{score_type}")) |>
+   tidyr::pivot_wider(
+    names_from = score_type, 
+    values_from = score
+  ) |>
+  dplyr::left_join(wide_category_df)
+```
+
+```{r, fig.height = 3, fig.width=12}
+ggplot(combined_wide_df) + 
+  aes(x = estimate_score_immune, y = celltype_immune) + 
+  geom_point() + 
+  geom_smooth(method = "lm") +
+  facet_wrap(vars(project_id), scales = "free_y", nrow = 1) +
+  labs(
+    title = "Immune celltypes", 
+    x = "ESTIMATE immune score", 
+    y = "Immune cell fraction\nin consensus celltypes"
+  )
+```
+
+
+```{r, fig.height = 3, fig.width=12}
+ggplot(combined_wide_df) + 
+  aes(x = estimate_score_stromal, y = celltype_stromal) + 
+  geom_point() + 
+  geom_smooth(method = "lm") +
+  facet_wrap(vars(project_id), scales = "free_y", nrow = 1) +
+  labs(
+    title = "Stromal celltypes", 
+    x = "ESTIMATE stromal score", 
+    y = "Stromal cell fraction\nin consensus celltypes"
+  )
+```
+
+* Immune plots:
+  * SCPCP000001, SCPCP00006, and SCPCP000017 show a positive relationship between quantities
+  * There is no apparent relationship for SCPCP000002 or SCPCP000009
+* Stromal plots:
+  * SCPCP00006 shows a positive relationship between quantities, and SCPCP000017 _may_ be trending towards a significant positive relationship (but that's a fairly wide +/- SE)
+  * There is no apparent relationship for SCPCP000001, SCPCP000002, or SCPCP000009
+
+That said, SCPCP000006 has an outlier point that may be driving the observed positive relationships.
+
+
+### Relationship between other cell types and tumor purity
+
+In theory, the `Unknown` cells in our consensus annotation are more likely to be tumor cells.
+We also excluded several cell types from analysis which might also be tumor.
+Similarly in theory, the ESTIMATE score (sum of immune and stromal) is expected to be proportional to tumor purity.
```

Comment:
```suggestion
Similarly in theory, the ESTIMATE score (sum of immune and stromal) is expected to be proportional to tumor purity where higher ESTIMATE scores correspond to lower tumor purity. 
Therefore we expect there to be a negative correlation between fractions of cells that are non-immune/non-stromal and the ESTIMATE score. 
```
Let's just be explicit about what this metric is looking at since I think this concept can be confusing.

Reply:
Oooh indeed thank you, as I was confusing myself too! Negative indeed does make sense. phew.